Extension/package.json

```diff
@@ -154,6 +154,18 @@
           "minimum": 0.1,
           "maximum": 1
         },
+        "C_Cpp.inactiveRegionFontColor": {
```

Comment:
Should this be "inactiveRegionForegroundColor"? VS Code seems to use the foreground instead of font term: https://code.visualstudio.com/docs/getstarted/theme-color-reference .